README.md

```diff
@@ -1705,6 +1705,7 @@ A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an
 
 - [Fess](https://fess.codelibs.org/) - Fess is a very powerful and easily deployable Enterprise Search Server. ([Demo](https://search.n2sm.co.jp/), [Source Code](https://github.com/codelibs/fess)) `Apache-2.0` `Java/Docker`
 - [Gigablast](https://www.gigablast.com/) - Open-source search engine. ([Source Code](https://github.com/gigablast/open-source-search-engine)) `Apache-2.0` `C++`
+- [Hound](https://github.com/hound-search/hound) `⚠` - Lightning fast code searching made easy. `MIT` `Go/Docker`
```

Comment:
It is apparently not limited to Github repositories, and can index various types of repos (git, svn, mercurial...) regardless of the forge/host being used (https://github.com/hound-search/hound/blob/main/config-example.json#L11). I suggest removing the `⚠` symbol.

Reply:
I see, I was being proactive due to the first thing that was shown is Github. I will remove that